test-files/cases/attribute/unifying/highway-2888/Expected.osm

```diff
@@ -4709,6 +4709,55 @@
         <tag k="hoot:status" v="1"/>
         <tag k="hoot:id" v="-1"/>
     </node>
+    <way visible="true" id="-1000" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-111"/>
+        <nd ref="-109"/>
+        <nd ref="-108"/>
+        <nd ref="-107"/>
+        <nd ref="-135"/>
+        <nd ref="-1035"/>
+        <tag k="length" v="141.8"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-1000"/>
+    </way>
+    <way visible="true" id="-984" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-1107"/>
+        <nd ref="-1106"/>
+        <tag k="length" v="54.3"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-984"/>
+    </way>
+    <way visible="true" id="-982" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-1093"/>
+        <nd ref="-1050"/>
+        <tag k="length" v="279.6"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-982"/>
+    </way>
```

Comment:
Way `-982` is another dog-leg situation similar to the one above.
![2888-way-982](https://user-images.githubusercontent.com/13385275/53367247-3c43a380-390b-11e9-8be3-43624e02c879.png)

Reply:
This dog-leg is fixed with the new code.